modules/photons_control/clean.py

```diff
@@ -0,0 +1,183 @@
+from photons_control.planner import Skip
+
+from photons_messages import LightMessages
+
+from delfick_project.norms import sb, Meta
+
+from datetime import timedelta
+from humanize import naturaldelta, precisedelta
+
+
+def SetCleanConfig(indication, duration, reference=None, **kwargs):
+    """
+    Returns a valid message that will set the default clean cycle configuration
+    for the device.
+
+    For example:
+
+    .. code-block:: python
+
+        await target.send(SetCleanConfig(indication=True), ["d073d5000001", "d073d5000001"])
+    """
+
+    async def gen(ref, sender, **kwargs):
+        r = ref if reference is None else reference
+
+        plans = sender.make_plans("capability")
+        async for serial, _, info in sender.gatherer.gather(plans, r, **kwargs):
+            if info["cap"].has_hev:
+                yield LightMessages.SetHevCycleConfiguration(
+                    indication=bool(indication),
+                    duration_s=duration,
+                    target=serial,
+                    ack_required=True,
+                    res_required=False,
+                )
+
+    return FromGenerator(gen)
+
+
+@task
+class set_clean_config(task.Task):
+    """
+    Set the default clean cycle configuration for a device
+
+    ``target:clean_config match:cap=hev -- '{"indication": true, "duration": 7200}'``
+
+    It takes an "indication" field that is a boolen which indicates whether to
+    run a short flashing indication at the end of the HEV cycle. It also takes
+    a ``duration`` field that is the seconds of the duration.
+    """
+
+    target = task.requires_target()
+    reference = task.provides_reference(special=True)
+
+    async def execute_task(self, **kwargs):
+        options = self.photons_app.extra_as_json
+        options = sb.set_options(
+            indication=sb.required(sb.boolean()),
+            duration=sb.required(sb.integer_spec()),
+        ).normalise(Meta.empty(), options)
+
+        await self.target.send(SetCleanConfig(**options), self.reference, **kwargs)
+
+
+@task
+class get_clean_config(task.Task):
+    """
+    Get the default HEV cleaning cycle configuration values for the specified devices
+    """
+
+    target = task.requires_target()
+    reference = task.provides_reference(special=True)
+
+    async def execute_task(self, **kwargs):
+        async with self.target.session() as sender:
+            plans = sender.make_plans("hev_config")
+            async for serial, _, info in sender.gatherer.gather(plans, self.reference, **kwargs):
+                if info is not Skip:
+                    indicator = "yes" if info["indication"] else "no"
+                    duration = naturaldelta(timedelta(seconds=info["duration"]))
+                    print(serial)
+                    print(f"    Clean cycle duration: {duration}")
+                    print(f"    Flash when cycle ends: {indicator}")
+                    print("")
+
+
+def ChangeCleanCycle(enable, duration=0, **kwargs):
```

Comment:
```suggestion
def ChangeCleanCycle(*, enable, duration=0, **kwargs):
```

Make it so that enable has to be a keyword. `ChangeCleanCycle(enable=False)` reads a bit better than `ChangeCleanCycle(False)`